extensions/barrage/src/main/java/io/deephaven/extensions/barrage/util/ArrowToTableConverter.java

```diff
@@ -81,8 +85,25 @@ public synchronized void setSchema(final byte[] ipcMessage) {
         parseSchema((Schema) mi.header.header(new Schema()));
     }
 
+    /**
```

Comment:
I don't think it's appropriate as javadoc; this is more of an implementation note. I would move these inside the method.